R/calcOutput.R

```diff
@@ -97,6 +98,7 @@ calcOutput <- function(type,aggregate=TRUE,file=NULL,years=NULL,round=NULL,suppl
     # rename aggregate arguments from old to new convention, if necessary
     if(toupper(aggregate)=="GLO") aggregate <- "global"
     if(toupper(gsub("+","",aggregate,fixed = TRUE))=="REGGLO") aggregate <- "region+global"
+    if(toupper(gsub("+","",aggregate,fixed = TRUE))=="REGGLOEXTRA") aggregate <- "region+global+extra"
```

Comment:
please do not introduce this alternative syntax. For REGGLO it is just in there to be backwards compatible.

Reply:
See above.